packages/picasso.js/src/web/renderer/canvas-renderer/canvas-renderer.js

```diff
@@ -160,8 +203,15 @@ export function renderer(sceneFn = sceneFactory) {
       patterns = patternizer(el.ownerDocument);
     }
 
-    const g = el.getContext('2d');
+    const g = (buffer && buffer.getContext()) || el.getContext('2d');
     const dpiRatio = dpiScale(g);
+    const transform = buffer && settings.transform();
+    if (transform) {
+      el.width = el.width; // eslint-disable-line
```

Comment:
why is this done? (maybe add a comment)

Reply:
It's to clear the canvas. I've now added a comment in the code about it.

I did read up a little on this approach of clearing the canvas, and it seems it's not the optimal way to do it... But it's done like this in other places in the canvas-renderer, so I don't want to fiddle around with that now and risk anything breaking in this PR.

It seems like a more proper and better performant way of clearing a canvas would be to use `content.clearRect(0, 0, canvas.width, canvas.height)` instead... But it could have some side effects changing to that approach straight away...